plugins/UM3NetworkPrinting/src/Models.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+from typing import Optional
+
+
+class BaseModel:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+    def __eq__(self, other):
+        return self.__dict__ == other.__dict__ if type(self) == type(other) else False
+
+
+##  Represents an item in the cluster API response for installed materials.
+class ClusterMaterial(BaseModel):
+    def __init__(self, **kwargs):
```

Comment:
Missing typing -> None

Reply:
There's not real other way to fix it right now as we're stuck on Python 3.5. I don't like it 100% either, but this PR is already a HUGE improvement over the old situation.

Reply:
Explicitly write out the arguments and type them. I know it is not an ideal solution but it will allow mypy to check the call sites for the models.

Reply:
I'll do that for the only attributes we use then so far. The rest can be added later.